core/src/test/java/nl/nn/adapterframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -66,213 +93,214 @@ private void assertSenderException(SenderException ex) {
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?{param})");
-		sender.setUseNamedParams(true);
-		sender.configure();
-		sender.open();
+	@DatabaseTest
+	public void testUseNamedParametersWithoutParam(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?{param})");
+		fixedQuerySender.setUseNamedParams(true);
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		SenderException ex = assertThrows(SenderException.class, () -> sendMessage("dummy"));
+		SenderException ex = assertThrows(SenderException.class, () -> fixedQuerySender.sendMessage(new Message("dummy"), session));
 
-		assertSenderException(ex);
+		assertSenderException(databaseTestEnvironment.getDataSourceName(), ex);
 	}
 
-	@Test
-	public void testNamedParamInQueryFlagFalse() throws Exception {
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?{param})");
-		sender.setUseNamedParams(false);
-		sender.configure();
-		sender.open();
+	@DatabaseTest
+	public void testNamedParamInQueryFlagFalse(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?{param})");
+		fixedQuerySender.setUseNamedParams(false);
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		SenderException ex = assertThrows(SenderException.class, () -> sendMessage("dummy"));
+		SenderException ex = assertThrows(SenderException.class, () -> fixedQuerySender.sendMessage(new Message("dummy"), session));
 
-		assertSenderException(ex);
+		assertSenderException(databaseTestEnvironment.getDataSourceName(), ex);
 	}
 
-	@Test
-	public void testIncompleteNamedParamInQuery() throws Exception {
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?{param)");
-		sender.configure();
-		sender.open();
+	@DatabaseTest
+	public void testIncompleteNamedParamInQuery(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?{param)");
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		SenderException ex = assertThrows(SenderException.class, () -> sendMessage("dummy") );
+		SenderException ex = assertThrows(SenderException.class, () -> fixedQuerySender.sendMessage(new Message("dummy"), session));
 
-		assertSenderException(ex);
+		assertSenderException(databaseTestEnvironment.getDataSourceName(), ex);
 	}
 
-	@Test
+	@DatabaseTest
 	public void testMultipleColumnsReturnedWithSpaceBetween() throws Exception {
-		assumeThat(productKey, anyOf(is("H2"),is("Oracle")));
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?)");
-		sender.addParameter(new Parameter("param1", "value"));
+		assertThat(dataSourceName, anyOf(is("H2"), is("Oracle")));
```

Comment:
Hier stond assume niet assert :)

Reply:
Oeps, die heb ik ook gemist. Ik zag het wel staan, maar checkte niet de inhoud. Heb ze zelf nog niet eerder toegepast. 
De unittest slaagt blijkbaar nog wel gewoon... dus dan zou je bijna kunnen zeggen dat de assume niet nodig is.

Reply:
Ik heb hier inderdaad assert gebruikt, omdat we volgens mij met H2 en Oracle testen, sowieso H2.
Daarom vond ik de assume een beetje vreemd